config/locales/en.yml

```diff
@@ -142,6 +142,10 @@ en:
         sign_in: Sign in
       user:
         signed_in: ''
+  errors:
+    messages:
+      password:
+        password_strength: Passwords need to be stronger than that. Try a longer or more complicated password please
```

Comment:
bummer that the gem doesn't have a sensible default here!

Reply:
It actually looks like it does: https://github.com/bdmac/strong_password/blob/75a2c79df5c54adb97b79f1c630a3162370e022e/README.md#with-railsactivemodel-validations

> The default validation message is "is too weak". Rails will display a field of :password having too weak of a password with full_error_messages as "Password is too weak"